### DISCUSSION TITLE
I'm guessing it needs to be installed again after DSM update?

### DISCUSSION THREAD

**Discussion by simowilso** [007revad/ContainerManager_for_all_armv8#30] [Q&A]
This is a brilliant script, thank you, seems to work great on my DS418 so far. I do have the latest update pending for this box (the one that removes videostation etc. :( ). I assume I'll need to re-run the script again if I install it?

**Comment by 007revad** [comment]
It would depend on if the DSM update updates DSM's package limit file - like DSM 7.2.2 Update 2 does.

At the moment you'd need to uninstall Container Manager then run the script again.

I'll update the script so it can repair Container Manager if it is already installed and the package limit file has been changed by a DSM update.

**Reply by 007revad** [reply]
I just checked the script and it will already handle DSM updates. If Container Manager is already installed 
it checks if there's a newer version of Container Manager and asks if you want to update Container Manager.

So if a DSM update changes Container Manager to show it needs to be repaired: 
1. Do **NOT** click on Repair in package Center.
2. Just run the script again.
